src/ActiveLogin.Authentication.BankId.Api/Models/AuthRequest.cs

```diff
@@ -35,7 +35,7 @@ public AuthRequest(string endUserIp)
         /// The personal number of the user. 12 digits, century must be included (YYYYMMDDSSSC).
         /// If the personal number is excluded, the client must be started with the AutoStartToken returned in the response.
         /// </param>
-        public AuthRequest(string endUserIp, string personalIdentityNumber)
+        public AuthRequest(string endUserIp, string? personalIdentityNumber)
```

Comment:
Since we do not handle null case here, isn't it better to create additional constructor that does not take `personalIdentityNumber`?

Reply:
Agree, there shouldn't be a null thing there, good catch!